msws.sh

```diff
@@ -1,16 +1,21 @@
+state_file="$HOME/.last_msws"
+today="$(date +%Y-%m-%d)"
 forecast_url='https://api.weather.gov/gridpoints/MKX/38,64/forecast'
-curl_header='Content-Type: application/json'
+
+if [[ -f "$state_file" ]] && grep -q "$today" "$state_file"; then
+  echo "✅ Already posted today ($today). Skipping."
+  exit 0
+fi
 
 forecast="$(curl "$forecast_url")" 
 prob="$(jq '.properties.periods[0].probabilityOfPrecipitation.value' <<< "$forecast")"
 desc="$(jq '.properties.periods[0].detailedForecast' <<< "$forecast")"
-gif_message='
-  {"content": "https://tenor.com/view/there-is-a-storm-coming-weather-hurricane-tornado-gif-11678470"
-}'
-detail_message="{\"content\": $desc}"
```

Comment:
Were you thinking about setting the message contents through environment variables as well? $curl_header is also no longer defined.